pkg/docker/docker.go

```diff
@@ -151,3 +152,35 @@ func PushImage(ctx context.Context, log *zap.SugaredLogger, dryRun bool, image s
 
 	return nil
 }
+
+// ExtractDirectory copies the content from a directory out of the
+// container onto the host system
+func ExtractDirectory(ctx context.Context, log *zap.SugaredLogger, image string, dst string, src string) error {
```

Comment:
why `ExtraceDirectory`? I think `Copy` is more clear as a name.

Reply:
I started with a name that made sense to me, then found out about `docker cp` but decided that "Copy" was a bit vague. Apparently not.